pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
is there any way to prevent such things and validate what pnpm version during the install? IMHO this version shouldn't be "downgraded"

Reply:
Not sure about a permanent fix, but maybe we should at least update to the latest version. I did not dare to do this as I am not the pnpm expert that @shellscape is and it is a major update.

Reply:
we should be good with the latest pnpm major now, so we should keep the lock file at 5.4. please update locally to pnpm@7

Reply:
I updated the lock file again and also changed the pnpm version in package.json to latest. I think that is best indeed.